chromadb/db/impl/sqlite_pool.py

```diff
@@ -18,7 +18,7 @@ def __init__(
         self._pool = pool
         self._db_file = db_file
         self._conn = sqlite3.connect(
-            db_file, timeout=1000, check_same_thread=False, uri=is_uri, *args, **kwargs
+            db_file, check_same_thread=False, uri=is_uri, *args, **kwargs
```

Comment:
this parameter is in seconds and defaults to 5s
if there was a good reason for previously setting it to 16m, I'm happy to adjust this on a per-connection basis for vacuuming

(if something else is holding a write lock, SQLite will wait up to this long before failing the VACUUM command)

Reply:
ended up reverting this as it caused some slow tests to fail, I now set the timeout at time of vacuum